.github/workflows/linux.yml

```diff
@@ -4,33 +4,81 @@ on:
 
+env:
+  BASE_PACKAGES: binutils gcc g++ gfortran cmake python3 perl git git-lfs curl wget tar unzip build-essential
+
 jobs:
   build_uberenv_mode:
     name: Uberenv Build Mode (Linux)
     runs-on: ubuntu-latest
     steps:
     - name: Install Deps
       run: |
-           sudo apt-get update
-           sudo apt-get install binutils \
-                                gcc \
-                                g++ \
-                                gfortran \
-                                python3 \
-                                perl \
-                                git \
-                                git-lfs \
-                                curl \
-                                wget \
-                                tar \
-                                unzip \
-                                build-essential
-    - uses: actions/checkout@v2
+          sudo apt-get update
+          sudo apt-get install $BASE_PACKAGES
+    - uses: actions/checkout@v3
     - name: Run Uberenv
       run: |
-          cd  .ci/test-project
-          python3 ../../uberenv.py  --project-json=uberenv_config.json
+          cd .ci/test-project
+          python3 ../../uberenv.py --project-json=uberenv_config.json
           cat uberenv_libs/*.cmake
 
+  build_dev_build_full_mode:
+    name: Dev Build Full Mode (Linux)
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Deps
+      run: |
+          sudo apt-get update
+          sudo apt-get install $BASE_PACKAGES
+    - uses: actions/checkout@v3
+    - name: Run Uberenv
+      run: |
+          cp .ci/test-project/packages/magictestlib.tar.gz . && tar -xvf magictestlib.tar.gz && cp src/* .
+          cd .ci/test-project
+          python3 ../../uberenv.py --project-json=dev_build_config.json --spack-config-dir=spack_configs
+
+  build_install_full_mode:
+    name: Install Full Mode (Linux)
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Deps
+      run: |
+          sudo apt-get update
+          sudo apt-get install $BASE_PACKAGES
+    - uses: actions/checkout@v3
+    - name: Run Uberenv
+      run: |
+          cp .ci/test-project/packages/magictestlib.tar.gz . && tar -xvf magictestlib.tar.gz && cp src/* .
+          cd .ci/test-project
+          python3 ../../uberenv.py --project-json=install_config.json --spack-config-dir=spack_configs
 
+  build_dev_build_initconfig_mode:
+    name: Dev Build Initconfig Mode (Linux)
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Deps
+      run: |
+          sudo apt-get update
+          sudo apt-get install $BASE_PACKAGES
+    - uses: actions/checkout@v3
+    - name: Run Uberenv
+      run: |
+          cp .ci/test-project/packages/magictestlib.tar.gz . && tar -xvf magictestlib.tar.gz && cp src/* .
+          cd .ci/test-project
+          python3 ../../uberenv.py --project-json=dev_build_config.json --spack-config-dir=spack_configs --package-final-phase=initconfig
 
+  build_install_initconfig_mode:
```

Comment:
Can you add a full install test here as well?

Reply:
I do have a full install test https://github.com/LLNL/uberenv/blob/85c6e6762c3bb9b875e52cf22812d8bfe1206e62/.github/workflows/linux.yml#L41-L54

Reply:
I mean if you want to use truth and logic to prove your point :laughing: